external/fv3fit/tests/emulation/layers/test_fields.py

```diff
@@ -8,6 +8,7 @@
     IncrementedFieldOutput,
```

Comment:
My sense is that these tests may not be necessary since the Field objects are essentially private helpers of `MicrophysicsConfig` which is tested.